app/components/contact-page-components/SendMessageForm.tsx

```diff
@@ -0,0 +1,64 @@
+import { RefObject, SetStateAction, FC, FormEvent, Dispatch } from "react";
+
+import FormInput from "./FormInput";
+import Button from "../Button";
+import { inputFields } from "@/constants";
+
+interface SendMessageFormProps {
+  form: RefObject<HTMLFormElement>;
+  sendEmail: (e: FormEvent<HTMLFormElement>) => void;
+  removeErrorMessage: (errorMsg: string) => void;
+  errorList: string;
+  formData: {
+    user_name: string;
+    user_email: string;
+    message: string;
+    contact_info: string;
+  };
+  setFormData: Dispatch<
+    SetStateAction<{
+      user_name: string;
+      user_email: string;
+      message: string;
+      contact_info: string;
+    }>
+  >;
+}
+const SendMessageForm: FC<SendMessageFormProps> = ({
+  form,
+  sendEmail,
+  removeErrorMessage,
+  errorList,
+  formData,
+  setFormData,
```

Comment:
When you have multiple instances of formData.someField, it might be more readable and less repetitive to destructure the properties you need.

```
const SendMessageForm: FC<SendMessageFormProps> = ({
  form,
  sendEmail,
  removeErrorMessage,
  errorList,
  formData: { user_name, user_email, message, contact_info },
  setFormData,
}) => {
```